CHANGELOG/CHANGELOG-1.24.md

```diff
@@ -659,7 +659,7 @@ Service `ClusterIP` are unique, hence, trying to create a Service with a `Cluste
 ### Deprecation
 
 - Deprecated `Service.Spec.LoadBalancerIP`. This field was under-specified and its meaning varies across implementations.  As of Kubernetes v1.24, users are encouraged to use implementation-specific annotations when available.  This field may be removed in a future API version. ([#107235](https://github.com/kubernetes/kubernetes/pull/107235), [@uablrek](https://github.com/uablrek))
-- Kube-apiserver: the `--master-count` flag and `--endpoint-reconciler-type=master-count` reconciler are deprecated in favor of the lease reconciler ([#108062](https://github.com/kubernetes/kubernetes/pull/108062), [@aojea](https://github.com/aojea))
+- Kube-apiserver: the `--apiserver-count` flag and `--endpoint-reconciler-type=master-count` reconciler are deprecated in favor of the lease reconciler ([#108062](https://github.com/kubernetes/kubernetes/pull/108062), [@aojea](https://github.com/aojea))
```

Comment:
I don't think the CHANGELOG should be edited by hand. The change should come as a release note to the next 1.24 patch release.